langserver/handlers/handlers_test.go

```diff
@@ -16,29 +17,38 @@ import (
 	"github.com/stretchr/testify/mock"
 )
 
-const initializeResponse = `{
-	"jsonrpc": "2.0",
-	"id": 1,
-	"result": {
-		"capabilities": {
-			"textDocumentSync": {
-				"openClose": true,
-				"change": 2
-			},
-			"completionProvider": {},
-			"documentSymbolProvider":true,
-			"documentFormattingProvider":true,
-			"executeCommandProvider": {
-				"commands": ["rootmodules"]
+func initializeResponse(t *testing.T, commandSuffix string) string {
```

Comment:
Assuming that the ID is an optional parameter, should we maybe have a dedicated test which is the only one that defines it, instead of changing the default test data which used throughout other tests?

Relatedly I think we should have tests which do _not_ set that parameter, because that will at this moment be any client other than VSCode and we intend to continue supporting these clients.

Reply:
yes I agree I will change the default